nextflow.config

```diff
@@ -204,6 +211,7 @@ profiles {
     liftover_test  { includeConfig 'conf/tests/liftover_test.config'  }
     liftover_truth { includeConfig 'conf/tests/liftover_truth.config' }
     test_full      { includeConfig 'conf/test_full.config'            }
+    test_happy     { includeConfig 'conf/tests/test_happy.config'     }
     test_somatic_full { includeConfig 'conf/test_full_somatic.config' }
```

Comment:
Some tests are in directory test, some arent. Why not make it consistent?
On a side note: Harshil alignment?

Reply:
I wanted to make test and test_full more visible. That is the only reason I keep it just under assets